spec/repositories/valkyrie/file_repository_spec.rb

```diff
@@ -2,7 +2,7 @@
 require 'rails_helper'
 
 RSpec.describe Valkyrie::FileRepository do
-  let(:repository) { instance_double(Valkyrie::FileRepository::Memory) }
+  let(:repository) { Valkyrie::FileRepository::Memory.new }
```

Comment:
Just checking -- we're using a real instance here because of the "memory adapter is the reference implementation/documentation" decision and want to validate it?